src/routes/settings/components/Settings.js

```diff
@@ -0,0 +1,197 @@
+// @flow
+
+import * as React from 'react'
+import { SectionList, StyleSheet, Switch, AsyncStorage, Linking } from 'react-native'
+import styled from 'styled-components/native'
+
+import SettingItem from './SettingItem'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+import type { NavigationScreenProp } from 'react-navigation'
+import { reduce } from 'lodash/collection'
+import type { TFunction } from 'react-i18next'
+
+type PropsType = {
+  navigation: NavigationScreenProp<*>,
+  theme: ThemeType,
+  language: string,
+  t: TFunction
+}
+
+type SettingsType = {|
+  errorTracking: boolean,
+  test: boolean
+|}
+
+type StateType = {
+  settings: SettingsType,
+  settingsLoaded: boolean
+}
+
+type ItemType = {
+  title: string, description: string,
+  hasSwitch?: true,
+  onPress?: () => void
+}
+
+type SectionType = { title: ?string }
+
+const ItemSeparator = styled.View`
+    background-color: ${props => props.theme.colors.textDecorationColor};;
+    height: ${StyleSheet.hairlineWidth};
+`
+
+const SectionHeaderContainer = styled.View`
+`
+
+const SectionHeader = styled.Text`
+    padding: 20px;
+    color: ${props => props.theme.colors.textColor};
+`
+
+export default class Settings extends React.Component<PropsType, StateType> {
+  sections = () => {
+    const {t, language} = this.props
+
+    return ([
+      {
+        title: 'Placeholder',
+        data: [
+          {
+            title: 'Placeholder',
+            description: 'Placeholder'
+          },
+          {
+            title: 'Placeholder',
+            description: 'Placeholder'
+          }
+        ]
+      },
+      {
+        data: [
+          {
+            title: t('troubleshooting'),
+            description: t('troubleshootingDescription'),
+            hasSwitch: true,
+            onPress: () => { this.toggleErrorTracking() }
+          },
+          {
+            title: t('about'),
+            onPress: () => {
+              if (language === 'de') {
+                Linking.openURL('https://integreat-app.de/')
+              } else {
+                Linking.openURL('https://integreat-app.de/en/')
+              }
+            }
+          },
+          {
+            title: t('privacyPolicy'),
+            onPress: () => {
+              if (language === 'de') {
+                Linking.openURL('https://integreat-app.de/datenschutz-webseite/')
+              } else {
+                Linking.openURL('https://integreat-app.de/en/privacy-website/')
```

Comment:
We might want to extract all these urls to some constants file

Reply:
We could not as these are not reused I see not really a point in doing it.